routes/server.php

```diff
@@ -18,13 +18,14 @@
 |
 */
 Route::group(['prefix' => 'settings'], function () {
+    Route::get('/', 'Settings\SettingsController@index')->name('server.settings');
     Route::get('/allocation', 'Settings\AllocationController@index')->name('server.settings.allocation');
-    Route::get('/name', 'Settings\NameController@index')->name('server.settings.name');
```

Comment:
This is a breaking change to anyone who might be reliant on that name elsewhere in their code. Please include backwards support for this, and update the readme to include a deprecation notice on that name. :)

Reply:
I've pushed it. I'm not sure if I included the deprecation notice correctly so please review that 👍